pom.xml

```diff
@@ -44,7 +44,7 @@
             <dependency>
                 <groupId>com.redhat.lightblue.rest</groupId>
                 <artifactId>lightblue-rest-integration-test</artifactId>
-                <version>2.28.0</version>
+                <version>2.29.0-SNAPSHOT</version>
```

Comment:
Is this using a snapshot until we "release" LB rest?

Reply:
Yes